lib/collection/src/operations/cluster_ops.rs

```diff
@@ -92,7 +86,6 @@ pub struct DropShardingKey {
 pub struct RestartTransfer {
     pub shard_id: ShardId,
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    #[schemars(skip)] // TODO(resharding): expose once we release resharding
     pub to_shard_id: Option<ShardId>,
```

Comment:
Same. Users _probably_ don't have to know about this.

Reply:
I think it's fine to hide indeed. It's easier to unhide them later than the other way around.